tensorflow/contrib/lite/examples/python/label_image.md

```diff
@@ -0,0 +1,35 @@
+With model (mobilenet_v1_1.0_224_quant.tflite), input image
```

Comment:
Thanks for the documentation on how to use it. A few small requests.

Can you start off with something like "Documentation on how to use `label_image.py` to run inference on a single image on an image model."

Additionally, could you add a link to the model files you used (both here and below). If the image you mentioned and labels file are also available within the TensorFlow repository then please add links to those add well to make it easier for others to reproduce.

Reply:
done